src/store/CategoricalDisplayStore.js

```diff
@@ -401,6 +401,16 @@ export default class CategoricalDisplayStore {
         if (!this.mainStore.chart.isMobile) {
             setTimeout(() => this.searchInput.current.focus(), 0);
         }
+
+        if (!this.mainStore.chart.isMobile) {
+            const categories = Object.keys(this.categoryElements);
+            const filtered_categories = categories.filter(item => this.categoryElements[item] !== null);
+            const last_category = this.filteredItems?.slice(-1)[0].categoryId;
+            const last_category_bottom_gap = this.height - (64 + (filtered_categories.length - 1) * 40); // to make the last category height reach it's filter tab
+            if (this.categoryElements[last_category]) {
+                this.categoryElements[last_category].style.minHeight = `${last_category_bottom_gap}px`;
+            }
+        }
```

Comment:
I removed this piece of code earlier because it was creating some unwanted gap between the categories

Reply:
It add the min_height to the last market symbols div inorder to fix scrolling issue. when the markets object change and some of them get removed,  it gets collapsed , I used `**last_category = this.filteredItems?.slice(-1)[0].categoryId;**` to fix this issue and always add min_height to visible last category.

Reply:
@balakrishna-binary can you share the PR where you made changes to this piece of code. This might have fixed the scroll issue perhaps.

Reply:
@akmal-binary PR - https://github.com/binary-com/SmartCharts/pull/1192/files. But I'm not sure if it fixes this issue. I'll add a regression QA checklist to make sure the extra space issue is not introduced again,